src/dspbase.jl

```diff
@@ -794,13 +794,13 @@ function conv(u::AbstractVector{T}, v::Transpose{T,<:AbstractVector}, A::Abstrac
     if any(conv_axis_with_offset, (axes(u)..., axes(v)..., axes(A)...))
         throw(ArgumentError("offset axes not supported"))
     end
-    m = length(u)+size(A,1)-1
-    n = length(v)+size(A,2)-1
+    m = length(u) + size(A, 1) - 1
+    n = length(v) + size(A, 2) - 1
     B = zeros(T, m, n)
-    B[1:size(A,1),1:size(A,2)] = A
-    u = fft([u;zeros(T,m-length(u))])
-    v = fft([v transpose(zeros(T,n-length(v)))])
-    C = ifft(fft(B) .* (u * v))
+    B[CartesianIndices(A)] = A
+    u, vt = fft.(_zeropad.((u, transpose(v)), (m, n)))
```

Comment:
I think this is easier both on the compiler and for anyone reading the code if done more explicitly instead of broadcasting:
```suggestion
    u = fft(_zeropad(u, m))
    vt = fft(_zeropad(transpose(v), n))
```